Bibliotheque nationale de France.js

```diff
@@ -9,7 +10,7 @@
     "translatorType": 4,
     "browserSupport": "g",
     "lastUpdated": "2016-02-18 11:28:15"
-    }
+     }
```

Comment:
Delete the spaces before the ending bracket here (this is essential to automatically recognize the ending of the metadata block) and make sure that the other things are indented correctly, i.e.
```js
{
    "translatorID": "47533cd7-ccaa-47a7-81bb-71c45e68a74d",
    "label": "Bibliothèque nationale de France",
    "creator": "Florian Ziche, Sylvain Machefert",
    "target": "^https?://[^/]*catalogue\\.bnf\\.fr",
    "minVersion": "3.0",
    "maxVersion": "",
    "priority": 100,
    "inRepository": true,
    "translatorType": 4,
    "browserSupport": "g",
    "lastUpdated": "2016-02-18 11:28:15"
}
```